### DISCUSSION TITLE
Add the ability to set a "default user group" when a user is added through the control panel, not just public registration

### DISCUSSION THREAD

**Discussion by darylknight** [craftcms/cms#11178] [Ideas]
Context: We typically set up our sites with a single user group, who are allowed to edit / create / delete everything - we usually don't need anything more granular than that. We set up that user group's permissions so that they're able to add users to user groups, but can't assign granular permissions.

During Craft training for clients, I explicitly call out the fact that when they add new users, they also have to go to the permissions tab and check the box for the new user to assign them to the Site Admin user group. A few months down the line, they rarely remember (which is understandable, given that permissions is under a separate tab), then they email us asking why the new user isn't able to login to the control panel.

This has been an issue with clients and our own team for months / years. There is a setting to put users who register through the front end in a user group by default, but would it be possible to set a default user group for users created through the control panel?

I'm not sure what the downsides would be - this could be left empty if people didn't want to use it - and it would make a lot more sense for the larger user base who just want to "add a new editor to the site".

![image](https://user-images.githubusercontent.com/1250017/167641882-a596748c-77de-46de-8952-239abbfa9d04.png)

**Comment by brandonkelly** [comment]
Will have to think about this. If they are going to forget a step in user registration, it would be better if it were to grant additional permissions, rather than to take away default permissions.

**Reply by darylknight** [reply]
Yeah, I understand. I'm just finding that the reality is that they never remember to do anything with permissions at all, and they find it rather counter-intuitive that when they want to "add a new user to the CMS", the new user isn't able to do anything.

I'd suggest that either
1, an optional default user group might be useful
2, if not - perhaps a baked in note or ability to add a UI note underneath the Email field saying "Remember to assign user permissions under the Permissions tab, otherwise the new user won't be able to do anything"

![image](https://user-images.githubusercontent.com/1250017/167787658-9c8bbf55-e400-4cb1-b65f-82b1ea4f9be1.png)
